devtools/platform-descriptor-json/src/main/resources/templates/dockerfile-jvm.ftl

```diff
@@ -14,21 +14,28 @@
 # docker run -i --rm -p 8080:8080 quarkus/${project_artifactId}-jvm
 #
 ###
-FROM fabric8/java-alpine-openjdk8-jre:1.6.5
-ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
-ENV AB_ENABLED=jmx_exporter
+ARG JAVA_PACKAGE=java-1.8.0-openjdk-headless
+ARG RUN_JAVA_VERSION=1.3.5
+
+# Install java and the run-java script
+# Also set up permissions for user `1001`
+RUN microdnf install ${JAVA_PACKAGE} \
+    && microdnf clean all \
+    && mkdir /deployments \
+    && chown -R 1001 /deployments \
+    && chmod -R "g+rwX" /deployments \
+    && chown -R 1001:root /deployments \
+    && curl http://repo1.maven.org/maven2/io/fabric8/run-java-sh/${RUN_JAVA_VERSION}/run-java-sh-${RUN_JAVA_VERSION}-sh.sh > /deployments/run-java.sh \
+    && chmod 755 /deployments/run-java.sh \
+    && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/lib/security/java.security
 
-# Be prepared for running in OpenShift too
-RUN adduser -G root --no-create-home --disabled-password 1001 \
-  && chown -R 1001 /deployments \
-  && chmod -R "g+rwX" /deployments \
-  && chown -R 1001:root /deployments
+ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
 
 COPY ${build_dir}/lib/* /deployments/lib/
 COPY ${build_dir}/*-runner.jar /deployments/app.jar
-EXPOSE 8080
+ENTRYPOINT [ "/deployments/run-java.sh" ]
 
-# run with user 1001
+EXPOSE 8080
 USER 1001
 
 ENTRYPOINT [ "/deployments/run-java.sh" ]
```

Comment:
You have 2 entry points?

Reply:
Sorry, bad copy and paste, it's been fixed.